snow/consensus/snowball/tree_test.go

```diff
@@ -875,9 +814,10 @@ func TestSnowballFilterBinaryChildren(t *testing.T) {
 
 	{
 		expected := `SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 0
-    SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 2
-        SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = true)) Bits = [3, 256)
-        SB(PreferenceStrength = 0, SF(Confidence = 0, Finalized = false)) Bits = [3, 256)
+    SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = false)) Bits = [1, 2)
```

Comment:
See graphs added in https://github.com/ava-labs/avalanchego/pull/2949/commits/565c5b1a932e66070504b7f42722ea40f3de0789#diff-1544dc37c28e958d7c8c249b181cc9fadda8acae042a933112e3535284d1efc8 for visualization